src/tasks/push-components.js

```diff
@@ -58,7 +58,7 @@ const getDataFromPath = async (path) => {
 const createContentList = (content, key) => {
   if (content[key]) return content[key]
   else if (Array.isArray(content)) return [...content]
-  else return [content]
+  else return []
```

Comment:
I commented to do what [you did above](https://github.com/storyblok/storyblok-cli/pull/53/files#diff-a442a9eaf8e2cf50032b0ebe2fd90c6115624fdae54bd8f965e64af7ca2e628eR48), however in this file `src/tasks/push-components.js`